### PR TITLE
test

### PR DESCRIPTION
**Pull request by denimozh**
test

### CODE REVIEW

File: src/utils/mongo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+let cached = global.mongoose
+
+if (!cached) {
+  cached = global.mongoose = { conn: null, promise: null }
+}
+
+async function dbConnect() {
+  if (cached.conn) {
+    return cached.conn
```
Comment (AliakseiDeboi) [inline src/utils/mongo.js]: Please, use ;

File: src/utils/mongo.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
+console.log("123")
```
Comment (AliakseiDeboi) [inline src/utils/mongo.js]: please, remove this console.log

**Review by AliakseiDeboi — CHANGES_REQUESTED** [review]

File: src/utils/mongo.js (hunk 2 above)
Comment (denimozh) [reply src/utils/mongo.js]: fixed

**Review by AliakseiDeboi — APPROVED** [review]
LGTM

**Pull request merged**